db/facade.py

```diff
@@ -0,0 +1,61 @@
+"""Database Facade."""
+# import boto3
```

Comment:
The DynamoDb specific code should be refactored into its own module. But we can keep it here for now so that we can get this pull request done and start working calling it in other modules.

Reply:
Gotcha. That's the next thing to work on.